src/main/java/com/snowplowanalytics/redash/model/datasource/Options.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2018 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.redash.model.datasource;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Options {
+    private String host;
+    private int port;
+    private String user;
+    private String password;
+    @SerializedName("dbname")
+    private String dbName;
```

Comment:
Do these variables ever need to be changed after creation? If not please make them final.